creator-node/src/utils/validateAudiusUserMetadata.ts

```diff
@@ -39,13 +39,13 @@ export const validateAssociatedWallets = (metadataJSON: MetadataJson) => {
   return true
 }
 
-export const validateMetadata = (req: Request, metadataJSON: MetadataJson) => {
+export const validateMetadata = (_req: Request, metadataJSON: MetadataJson) => {
```

Comment:
bad practice to be naming function parameters with underscore
can you just do as CustomRequest in the signature and not need to re-assign?

Reply:
@SidSethi I think it has to be `Request` in the function parameters because that's what express needs the type signature to be. We don't tell express or typescript anywhere that we injected custom fields so it gets confused. I'd been googling a bit how to get around this but in the meantime, we do have to do hacky stuff like
```
const customReq = req as CustomRequest
```